angular/src/lib/list-item/list-item.component.ts

```diff
@@ -102,6 +111,7 @@ export class ListItemComponent implements OnInit, AfterViewInit, AfterContentChe
       'cui-list-item' +
       `${(this.type && ` cui-list-item--${this.type}`) || ''}` +
       `${(this.active && ` active`) || ''}` +
+      `${(this.focus && ` focus`) || ''}` +
```

Comment:
I used @HostBinding before. It's a good example for using it here. Now I use :host under @Component. Maybe you can update the code.

Reply:
@ArthusLiang Yes, we want to change the way we do @HostBinding. This PR was about the SelectComponent in which I did change the style of HostBinding. If I change it in list-item, then I should change it in list and list-item-section. I think we should allocate a story to audit through the already existing components to make these adjustments.

Reply:
I went ahead and changed the Hostbinding of classNames for List and ListItem.